app/views/viewall/alumni.html.erb

```diff
@@ -1,4 +1,5 @@
 <h2 align="center">Complete Alumni Database</h2>
+<h4 align="center">Total Count: &nbsp; <label class="label label-info"> <%= @allAlums.count %> </label></h4>
```

Comment:
@aniarya82 
Sorry, what I meant was that the UI of badges and the color of `info` is not good.

So, please change this to plain numbers. We don't need a label here.